src/server_manager/cloud/gcp_api.ts

```diff
@@ -288,6 +321,22 @@ export class RestApiClient {
     await this.computeEngineOperationRegionWait(region, operation.name);
   }
 
+  /**
+   * Lists all static IPs in a project.
+   *
+   * @see https://cloud.google.com/compute/docs/reference/rest/v1/instances/list
+   *
+   * @param zone - Indicates the GCP project and zone.
+   * @param filter - See documentation.
```

Comment:
Filter is missing.
Shouldn't we only list the outline ips?

Reply:
Oops, docstring error.  (This method has now been removed.)